eth2/utils/tree_hash_derive/tests/tests.rs

```diff
@@ -1,179 +0,0 @@
-use cached_tree_hash::{CachedTreeHash, TreeHashCache};
```

Comment:
I think we can add these tests back. I will follow up on this.

Reply:
Oh, I got mixed up. We can't add them back because `CachedTreeHash` is incomplete (I thought these were just `TreeHash` tests)